### PR TITLE
display locale and voice id option on speak command

### PR DESCRIPTION
**Pull request by ecruzado**
screen shot 2018-09-05 at 4 40 21 pm

### CODE REVIEW

**Comment by dmarvp** [comment]
@jperata @ecruzado I think we should change the text of the token option to not say "Alexa virtual device"  and just "virtual device".

**Comment by jperata** [comment]
@dmarvp Agree it already works if it's a google one, so it makes sense.

File: bin/bst-speak.ts [hunk 1]
```diff
@@ -15,11 +15,11 @@ Global.initializeCLI().then(
     () => {
         program
             .usage("[options] <utterance>")
+            .option("-l, --locale <locale>", "The locale to use for the virtual device (en-US, en-GB, de-DE, etc.)")
             .option("-t, --token <token>", `The token for interacting with your virtual Alexa device - get it here:
```
Comment (jperata) [inline bin/bst-speak.ts]: rename to virtual device

**Comment by dmarvp** [comment]
@jperata @ecruzado also, does this fix the other error specified on the ticket?

> Additionally, when running the speak command before setting the token, an error is shown when specifying the locale option.

**Comment by ecruzado** [comment]
could't reproduce the issue, I am asking john to elaborate

**Pull request merged**